packages/vaex-core/vaex/__init__.py

```diff
@@ -243,6 +243,20 @@ def from_astropy_table(table):
     return vaex.file.other.DatasetAstropyTable(table=table)
 
 
+def from_dict(data):
+    """Create an in memory dataset from numpy arrays, in contrast to from_arrays this keeps the order of columns intact
```

Comment:
Please update the Docstring to reflect what the function does. 
Since the latest release, we have a established way of writing the Docstrings. Please rebase your branch to master, and look at the other examples. 

Otherwise it looks good